benchmark-service/benchmark/static/css/autocomplete.css

```diff
@@ -0,0 +1,189 @@
+#autoComplete_wrapper {
+    position:relative;
+    display:inline-block;
+}
+#autoComplete {
+    position:relative;
+    padding:0 2rem 0 3.5rem;
+    height:2.1rem;
+    width:6rem;
+    font-size:1.2rem;
+    outline:0;
+    border-radius:3px;
+    border:.05rem solid;
+    /* color:rgba(255,255,255,0); */
+    color:rgba(0,0,0,1);
+    background-repeat:no-repeat;
+    background-size:1.2rem;
+    background-origin:border-box;
+    background-position:center;
+    transition:all .4s ease;
+    -webkit-transition:all -webkit-transform .4s ease;
+    text-overflow:ellipsis;
+}
+#autoComplete::placeholder {
+    color:rgba(192,192,192,0);
+    transition:all .3s ease;
+}
+#autoComplete:hover::placeholder {
+    color:rgba(192,192,192,0.3);
+    transition:all .3s ease;
+    -webkit-transition:all -webkit-transform .3s ease;
+}
+#autoComplete:focus::placeholder {
+    padding:.1rem .6rem;
+    font-size:1rem;
+    color:rgba(192,192,192,0.3);
+    transition:all .3s ease;
+    -webkit-transition:all -webkit-transform .3s ease;
+}
+#autoComplete::selection {
+    color:transparent;
+}
+#autoComplete:focus::selection {
+    color:#fff;
+    background-color:rgba(192,192,192,1);
+}
+#autoComplete:hover {
+    padding:0 2rem 0 3.5rem;
+    color:rgba(192,192,192,1);
+    height:3rem;
+    width:16.5rem;
+    background-size:1.7rem;
+    background-position:left 1.05rem top .7rem;transition:all .3s ease;
+    -webkit-transition:all -webkit-transform .3s ease;
+}
+#autoComplete:focus {
+    padding:0 2rem 0 3.5rem;
+    color:rgba(192,192,192,1);
+    height:3rem;
+    width:16.5rem;
+    border:.05rem solid #000000;
+    background-size:1.5rem;
+    background-position:left 1.05rem top .8rem;
+    box-shadow:rgba(192,192,192,0.1) 0 0 20px 5px;
+}
+#autoComplete_list {
+    position:fixed;
+    z-index:1000;
+    padding:0;
+    left:0;
+    right:0;
+    margin-top:0;
+    margin-left:auto;
+    margin-right:auto;
+    width:18rem;
+    transition:all .1s ease-in-out;
+    -webkit-transition:all;
+    -webkit-transform .1s ease;
+}
+.autoComplete_result {
+    margin:.15rem auto;
+    padding:.6rem;
+    max-width:280px;
+    border:.05rem solid #e3e3e3;
+    list-style:none;
+    text-align:left;
+    font-size:1.1rem;
+    color:#7b7b7b;
+    transition:all .1s ease-in-out;
+    background-color:#fff;
+}
+.autoComplete_result::selection {
+    color:rgba(#fff,0);
+    background-color:rgba(#fff,0);
```

Comment:
I don't think these are valid CSS rules. I believe `rgba` requires decimal numbers, not hexadecimal ones. If you want white, you can just write: `color: #fff`.